frontend/packages/ceph-storage-plugin/integration-tests-cypress/views/multiple-pool.ts

```diff
@@ -0,0 +1,76 @@
+import { NS } from '../utils/consts';
+
+// Storage class form
+export const createStorageClassButton: string = 'item-create';
+
+// Create storage class from
+export const provisionerDropdown: string = 'storage-class-provisioner-dropdown';
+export const expandPvcCheckbox: string = 'expand-pvc-checkbox';
+export const storagePoolDropdown: string = 'pool-dropdown-toggle';
+export const createNewPoolButton: string = 'create-new-pool-button';
+
+// Create new pool form
+export const titleLabel: string = 'modal-title';
+export const poolNameTextBox: string = 'new-pool-name-textbox';
+export const replicaDropdown: string = 'replica-dropdown';
+export const confirmAction: string = 'confirm-action';
+export const cancelAction: string = 'modal-cancel-action';
+
+// Pool status
+export const emptyStateBody: string = 'empty-state-body';
```

Comment:
Constants that are only referenced in one place, don't deserve to be declared as constants. We can directly use them inside the getters. The abstraction provided by `storagePool` object is good enough. We don't need to reference selectors from here if it's only done once.

Reply:
Agreed but what if new test cases are written in the future for the same feature. It always better to write a loosely coupled code.

Reply:
I agree with you, but I am not sure if we will have a use case for pool flows again. If it's required in other tests most of the time we will be creating pools via CLI as it's much easier( and less error-prone). Our constants folder is full of constants that are used in only one place. It'd be better if we refactor it into a constant later when we require it, as the chances of these being used again are quite slim.

Reply:
ack

Reply:
removed constants which are all used in one place, please review